src/appdistribution/requests.ts

```diff
@@ -0,0 +1,154 @@
+import * as _ from "lodash";
+import * as api from "../api";
+import * as utils from "../utils";
+import { Distribution } from "./distribution";
+import { FirebaseError } from "../error";
+
+// tslint:disable-next-line:no-var-requires
+const pkg = require("../../package.json");
+
+/**
+ * Proxies HTTPS requests to the App Distribution server backend.
+ */
+export class AppDistributionRequests {
```

Comment:
How about `AppDistributionClient`? Plural class names feel weird to me.

Reply:
Good call, updated.